### ISSUE TITLE
Chat box on Butter App

### ISSUE THREAD

**Issue by Glamorgal** [butterproject/butter-desktop#138]
What about adding chat system to Butter? So we can communicate with each other and make friends, or share everything. Waitting for your suggestions.

**Comment by Octopixell** [comment]
I'd say add it as a plugin later, only for people that want it.

**Comment by Geczy** [comment]
No

**Comment by prcastro** [comment]
I'm also against it. Hard to maintain and to moderate.

**Comment by Octopixell** [comment]
It would turn in one of those chatboxes you see on these anime websites.. Just a load of crap and no real conversation..

**Comment by Glamorgal** [comment]
Maybe you're right. It would turn to one of those chatboxes but we could give it a try. And pretty sure there will be a lot of people to moderate that chat. But it was just an idea. :)

**Comment by Octopixell** [comment]
@Glamorgal any idea is more than welcome ofcourse! :+1: 
Like I said, you could maybe make it as a plugin, people that would want it, could use it. But it'd have to be seperately maintained and definitely seperated from the core project.

**Comment by Glamorgal** [comment]
OK I got it :) Thank you!

**Comment by SrPatinhas** [comment]
@Glamorgal its a good idea if made well from scratch, but like @Octopixell said, it will be a plugin, maybe a good plugin to ask on app install? :p
but maybe it could have this features:
 - Group Chat
 - Friends request ( maybe with some integration? like trakt or so? it would simplify the need for servers)
 - Share seen movies / shows / episodes and rate?
 - share plugins/packages with friends (if they come to the app) 
 - several rooms
 - etc

This is just some ideas, but for this to work should be well done and well think of ;)

**Comment by kisPocok** [comment]
It is nice, but it moves away us from Butter app's goal. I suggest to keep our poor functionality as a strict rule. Anyway it would be nice if plugin system comes real!

**Comment by NoNameProvided** [comment]
+1 against it. Hard to maintain and to moderate. A chat is totally unrelated to Butter.

**Comment by cfacundo** [comment]
-1

**Comment by toinouu** [comment]
-1000

**Comment by Draxler** [comment]
:-1:  
No chaty on butter .. only watching and enjoying. There's tons of chater options out there which make posible for anyone to chat live while watching whatever on Butter (if that make sence).. 
No need for chat box to have Butter name on it ... And like other alrready said. This have nothing to do with Butter and chatting while watching have no sence.. For chat in general people have other sources for that ..

**Comment by mo5tlef** [comment]
@Draxler Right

**Comment by kisPocok** [comment]
I suggest to close this bug for now.

**Comment by SrPatinhas** [comment]
i see you all have no friends to share movies you like or see the real point to have this as a plugin, that would be as a plugin, not in the app itself, witch one of you would install it if you want. and im up for this plugin to share movies or shows or whatever with my friends.

@Glamorgal i will not close this because to avoid to be duplicated. but later will be pass to another repo of plugins.

**Comment by Octopixell** [comment]
@SrPatinhas are there any types of documentation or ideas we can read into about future plugin development?

**Comment by SrPatinhas** [comment]
@Octopixell soon we will have the documentation ;)

**Comment by Octopixell** [comment]
@SrPatinhas you Sir, are amazing, thanks! As soon as we can write plugins people can just make Butter what they want it to be, rather than have these types of discussions where people don't even bother to say something other than -1 or no :)

**Comment by mo5tlef** [comment]
@SrPatinhas Create a Simple application for movies, and easy to use, small size .. This is important and Good idea
( Notifications ) plugin is important more than Chat!

**Comment by Octopixell** [comment]
@mo5tlef I do hope you understand it wouldn't be the core team working on things like chat plugins and the likes. If it was important enough for the core team to work on it, it would be in the core. ;)

**Comment by mo5tlef** [comment]
@Octopixell Ok, Go head.
But u are wasting your time
Chating is everywhere
butter Must be ( Movies time )

**Comment by Draxler** [comment]
@SrPatinhas 

Suggesting that someone have no friends just because someone doesn't see much sence in chating while watching the movie is ridiculous at least. And second i already said that chat app doesn't have to have Butter name on it, which means theres a lot of others Chat sources outhere for you to use for exchange butter experiance. If you want to have this as a plug system, OK,  as long this is not intergrated into app it self because most of the people wouldn't touch this app if this jappened. People already see how anoying it is on Anime websites where you couldn't have a moment of piece out there without hearing this anoying sound of new messages and see these flashy chatbox on the side ...

**Comment by rufolangus** [comment]
No. If anything facebook integration. Like & share -> link opens butter to whatever they shared.

**Comment by vankasteelj** [comment]
why might be done, though, is adding our own comment section on official providers (for example: a disq instance), for people to comment the Open catalog. But bare in mind that butter relies heavily on providers, it's not Netflix, shipping its own content.

**Comment by xaiki** [comment]
as @vankasteelj and @SrPatinhas said this is a good plugin idea, let's keep it around, then butter builders will decide weather they want to ship this or not.

**Comment by rgamez9** [comment]
UNSUBSCRIBE 


Sent via the Samsung Galaxy S®6 active, an AT&T 4G LTE smartphone-------- Original message --------From: Jean van Kasteel <notifications@github.com> Date: 01/28/2016  5:43 PM  (GMT-06:00) To: butterproject/butter-desktop <butter-desktop@noreply.github.com> Subject: Re: [butter-desktop] Chat box on Butter App (#138) 
why might be done, though, is adding our own comment section on official providers (for example: a disq instance), for people to comment the Open catalog. But bare in mind that butter relies heavily on providers, it's not Netflix, shipping its own content.
